<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClemsonRegistrationBot</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0.937, y2:0.801136, stop:0 		rgba(173, 216, 230, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>10</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Clemson Registration Bot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgb(255, 229, 253);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#start_button{
	background-color: rgb(85, 170, 127);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton#start_button:pressed{
	background-color: rgb(69, 255, 162);
}</string>
    </property>
    <property name="text">
     <string>Start Bot</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_text">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="password_text">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_box">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="start_time_text">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>Start Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgb(255, 229, 253);
border-radius: 10px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="password_text_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>Pin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pin_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgb(255, 229, 253);
border-radius: 10px;</string>
    </property>
   </widget>
   <zorder>title</zorder>
   <zorder>start_button</zorder>
   <zorder>username_text</zorder>
   <zorder>password_text</zorder>
   <zorder>time_box</zorder>
   <zorder>start_time_text</zorder>
   <zorder>password_box</zorder>
   <zorder>username_box</zorder>
   <zorder>password_text_2</zorder>
   <zorder>pin_box</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
